source/develop/release-management/features/infra/aaa_faq.html.md

```diff
@@ -115,3 +115,18 @@ Yes, it's possible with *apache mod_auth_cas* module. Full blog post is availabl
 
 Unfortunatelly it's not supported, yet. You can track [this](https://bugzilla.redhat.com/show_bug.cgi?id=829292) bugzilla for more information.
 
+## Is it possible to use modify search filter for users?
+Yes it's possible with a little hack to you properties file.
+Please append following lines to your profile configuration:
+
+```
+sequence.simple-query-principals.005.description = modify filter to search only by uid
+sequence.simple-query-principals.005.type = regex
+sequence.simple-query-principals.005.regex.value = ${seq:filter}
+sequence.simple-query-principals.005.regex.flags = a
+sequence.simple-query-principals.005.regex.pattern = \\(givenName=[^)]*\\)|\\(sn=[^)]*\\)|\\(displayName=[^)]*\\)
+sequence.simple-query-principals.005.regex.replacement.filter = 
+```
+
+This example will replaces occurances of givenName, sn and displayName and it's values by empty string.
+So only uid persist in filter. You can modify it as you want just by changing the regex expression.
```

Comment:
Suggest:
"Only uid will persist in the filter. You can modify it as you like, simply by changing the regex expression.